add -wd308 compiler option to avoid compiler warning #308 being treated as an error
author: Kenneth Hoste (HPC-UGent)
-- Rtsne.orig/src/Makevars	2017-04-14 17:51:58.000000000 +0200
+++ Rtsne/src/Makevars	2017-07-07 15:29:41.426358000 +0200
@@ -25,4 +25,5 @@
 ## done by a number of packages, but recommended only for more advanced users
 ## comfortable with autoconf and its related tools.
 
-
+# disable Intel C++ compiler (icpc) warning/error #308, to avoid 'member "std::complex<double>::_M_value" is inaccessible'
+PKG_CXXFLAGS = -wd308
--- Rtsne.orig/MD5	2017-04-14 18:38:37.000000000 +0200
+++ Rtsne/MD5	2017-07-07 15:29:58.247065000 +0200
@@ -6,7 +6,7 @@
 7af8fde3597798501e00bdc337d8f03e *README.md
 42ae2f3c4872e03c5e401d949a86cad9 *inst/CITATION
 065bed604fc3d4be094fea9fbf7f162e *man/Rtsne.Rd
-fcd429500be5e9251157f001145e8a8b *src/Makevars
+5ed55ce6fb49190f62c48ff8ce8705a1 *src/Makevars
 e9a378d06e68a325926bf92e779f280e *src/Makevars.win
 5d991038a3c68e7232302c837179c26d *src/RcppExports.cpp
 3740407bdc49d16ee9a7a8d5ee96a327 *src/Rtsne.cpp
